apps/web/src/components/media-gallery/media-gallery.tsx

```diff
@@ -102,10 +102,10 @@ export default function MediaGallery({ media }: MediaGalleryProps) {
                 onClick={() => handleMediaChange(media)}
               >
                 <Image
-                  alt=""
+                  alt="media"
                   className={clsx(css.cover, css.fullWidth, css.rounded)}
                   layout="fill"
-                  src={media}
+                  src={`${media}?width=300`}
```

Comment:
this one still needs `sizes`. I _believe_ we use a 4-column grid for all screen sizes here so it might be as simple as 
 `sizes="25vw"`

Reply:
```suggestion
                  src={media}
                  sizes="25vw"
```